### PR TITLE
Root settings evaluated scope [WIP] 

### PR DESCRIPTION
**Pull request by lptr**

### CODE REVIEW

**Comment by lptr** [comment]
Quick feedback: https://builds.gradle.org/viewQueued.html?itemId=14310927

**Comment by lptr** [comment]
Next: https://builds.gradle.org/viewQueued.html?itemId=14311364

File: subprojects/core/src/main/java/org/gradle/execution/taskgraph/LocalTaskInfoExecutor.java [hunk 1]
```diff
@@ -21,14 +21,13 @@
 import org.gradle.api.internal.tasks.TaskExecutionContext;
 import org.gradle.api.internal.tasks.TaskStateInternal;
 import org.gradle.api.internal.tasks.execution.DefaultTaskExecutionContext;
-import org.gradle.internal.Factory;
 
 public class LocalTaskInfoExecutor implements WorkInfoExecutor {
     // This currently needs to be lazy, as it uses state that is not available when the graph is created
```
Comment (wolfs) [inline subprojects/core/src/main/java/org/gradle/execution/taskgraph/LocalTaskInfoExecutor.java]: Can we remove this comment now?

File: subprojects/core/src/main/java/org/gradle/internal/service/scopes/BuildScopeServices.java [hunk 2]
```diff
@@ -176,7 +175,6 @@
 public class BuildScopeServices extends DefaultServiceRegistry {
     public BuildScopeServices(final ServiceRegistry parent) {
         super(parent);
-        addProvider(new BuildCacheServices());
```
Comment (wolfs) [inline subprojects/core/src/main/java/org/gradle/internal/service/scopes/BuildScopeServices.java]: Does that mean we registered `BuildCacheServices` twice?

Comment (lptr) [reply subprojects/core/src/main/java/org/gradle/internal/service/scopes/BuildScopeServices.java]: I think it only meant that we were looking for `create*()` etc. methods, and found none.

**Comment by lptr** [comment]
Won't be pursuing this anymore.

**Pull request closed without merging**